README.md

```diff
@@ -87,58 +87,19 @@ Maintainers ([@open-telemetry/collector-contrib-maintainer](https://github.com/o
-| awsxrayexporter                | @kbrockhoff @anuraaga      |
-| azuremonitorexporter           | @pcwiese                   |
-| carbonexporter                 | @pjanotti                  |
-| elasticexporter                | @axw                       |
-| honeycombexporter              | @paulosman @lizthegrey     |
-| jaegerthrifthttpexporter       | @jpkrohling @pavolloffay   |
-| kinesisexporter                | @owais                     |
-| newrelicexporter               | @MrAlias                   |
-| sapmexporter                   | @owais @dmitryax           |
-| sentryexporter                 | @AbhiPrasad                |
-| signalfxexporter               | @pmcollins @asuresh4       |
-| splunkhecexporter              | @atoulme                   |
-| stackdriverexporter            | @nilebox @james-bebbington |
-
-### Receivers
-
-| Receiver                       | Reviewer(s)               |
-| ------------------------------ | ------------------------- |
-| awsecscontainermetricsreceiver | @kbrockhoff @anuraaga     |
-| awsxrayreceiver                | @kbrockhoff @anuraaga     |
-| carbonreceiver                 | @pjanotti                 |
-| collectdreceiver               | @owais                    |
-| dockerstatsreceiver            | @rmfitzpatrick            |
-| k8sclusterreceiver             | @asuresh4                 |
-| kubeletstatsreceiver           | @pmcollins @asuresh4      |
-| prometheusexecreceiver         | @keitwb @james-bebbington |
-| receivercreator                | @jrcamp                   |
-| redisreceiver                  | @pmcollins @jrcamp        |
-| sapmreceiver                   | @owais                    |
-| signalfxreceiver               | @pjanotti @asuresh4       |
-| simpleprometheusreceiver       | @asuresh4                 |
-| statsdreceiver                 | @keitwb @jmacd            |
-| wavefrontreceiver              | @pjanotti                 |
-
-### Processors
-
-| Processor                      | Reviewer(s)               |
-| ------------------------------ | ------------------------- |
-| groupbytrace                   | @jpkrohling               |
-| routing                        | @jpkrohling               |
-
-### Extensions
-
-| Extension                      | Reviewer(s)               |
-| ------------------------------ | ------------------------- |
-| httpforwarder                  | @asuresh4                 |
-| jmxmetrics                     | @rmfitzpatrick            |
-| observer                       | @asuresh4 @jrcamp         |
+## PRs and Reviews
+
+When creating a PR please following the process [described
+here](https://github.com/open-telemetry/opentelemetry-collector/blob/master/CONTRIBUTING.md#how-to-structure-prs-to-get-expedient-reviews).
+
+News PRs will be automatically associated with the reviewers based on
+[CODEOWNERS](.github/CODEOWNERS). PRs will be also automatically assigned to one of the
```

Comment:
Not part of this PR, but I noticed that the approvers group is also added as a reviewer to PRs. Is it possible to disable that?

Reply:
It should not be, we need one approval from that group as well.

Reply:
The approval from the components owner does not count as approval of the PR unfortunately unless we open permission to everyone to have write permission, which opens possibility to do releases etc.

Reply:
I had the impression that I started receiving notifications even for PRs that weren't for me to review, but I might be wrong. I'll leave a comment here if this happens.